sklearn/tests/test_naive_bayes.py

```diff
@@ -261,8 +263,8 @@ def test_discretenb_partial_fit(cls):
         assert_array_equal(clf1.feature_count_, clf3.feature_count_)
 
 
-@pytest.mark.parametrize('cls', [BernoulliNB, MultinomialNB, GaussianNB,
```

Comment:
You could rename the test `test_naive_bayes_pickle` and update the first comment. We want to check that pickling is working.

Reply:
Done in 6e51074